api/envoy/config/wasm/v2alpha/wasm.proto

```diff
@@ -0,0 +1,67 @@
+syntax = "proto3";
+
+package envoy.config.wasm.v2alpha;
+
+option java_outer_classname = "WasmProto";
+option java_multiple_files = true;
+option java_package = "io.envoyproxy.envoy.config.wasm.v2alpha";
+
+import "envoy/api/v2/core/base.proto";
+
+import "validate/validate.proto";
+
+// [#protodoc-title: Wasm service]
+
+// Configuration for a Wasm VM.
+// [#next-free-field: 6]
+message VmConfig {
+  // An ID which will be used along with a hash of the wasm code (or null_vm_id) to determine which
+  // VM will be used for the plugin. All plugins which use the same vm_id and code will use the same
+  // VM. May be left blank. Sharing a VM between plugins can reduce memory utilization and make sharing of data easier which may have security implications.  See ref: "TODO: add ref" for details.
+  string vm_id = 1;
+
+  // The Wasm runtime type (either "v8" or "null" for code comipled into Envoy).
+  string runtime = 2 {validate.rules).string = {min_bytes: 2}};
```

Comment:
nit: I would just make this min bytes 1 unless you want to actually make this require specific values

Reply:
Done.